exercism_config.gemspec

```
@@ -19,7 +19,10 @@ Gem::Specification.new do |spec|
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
   spec.bindir        = "bin"
-  spec.executables   = ["setup_exercism_config"]
+  spec.executables   = [
+    "setup_exercism_config",
+    "setup_local_aws",
```

```suggestion
    "setup_exercism_local_aws",
```

Let's rename this to be respectful to all the other gems fighting for the binstubs. 

Can you rename the file too pls?